### ISSUE TITLE
BUG - a11y  highlight colour does not have enough contrast with text

### ISSUE THREAD

**Issue by trallard** [Quansight-Labs/accessible-pygments#19]
## Issue

I just noticed in the PyData Sphinx theme where they are using the themes that the highlight colour for the dark a11y theme is preventing the text from being readable whether they are too bright or too dark and close to the foreground colour:

- a11y dark high contrast
![Blocks_—_PyData_Theme_documentation](https://user-images.githubusercontent.com/23552331/210867083-68c8fec1-09a8-42ed-8390-5bbfc76e3687.png)

- a11y light 
![Blocks_—_PyData_Theme_documentation](https://user-images.githubusercontent.com/23552331/210867131-2c79052d-52ec-45af-9f9c-ba3ffecdc92f.png)

Source for these screenshots https://pydata-sphinx-theme.readthedocs.io/en/latest/examples/kitchen-sink/blocks.html
